File: siserp-logistica-backend/src/main/java/com/besoft/siserp/lgtc/dto/request/CompraRegistrarDetalleRequest.java
package com.besoft.siserp.lgtc.dto.request;

import java.io.Serializable;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class CompraRegistrarDetalleRequest implements Serializable {
	private static final long serialVersionUID = 1L;

	private Long idProducto;
	private String nomProducto;
	private Double cantidad;
	private Double cantidadPerfecto;
	private Double cantidadDaniado;
	private Double precioUnitario;
	private Double subTotal;
}

File: siserp-logistica-frontend/src/main/siserp-logistica-frontend/src/app/modules/intranet/dto/request/compra-registrar-detalle.request.ts
export class CompraRegistrarDetalleRequest {
    id: number;
    idProducto: number;
    nomProducto: string;
    cantidad: number;
    cantidadPerfecto: number;
    cantidadDaniado: number;
    precioUnitario: number;
    subTotal: number;
}